app/src/main/java/com/framgia/soundclound/screen/local/TrackLocalViewModel.java

```diff
@@ -0,0 +1,96 @@
+package com.framgia.soundclound.screen.local;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.ContentResolver;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.database.Cursor;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.net.Uri;
+import android.provider.MediaStore;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.app.AppCompatActivity;
+
+import com.framgia.soundclound.BR;
+import com.framgia.soundclound.data.model.PublisherMetadata;
+import com.framgia.soundclound.data.model.Track;
+import com.framgia.soundclound.screen.main.MainActivity;
```

Comment:
ERROR: Unused import: com.framgia.soundclound.screen.main.MainActivity. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports